webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -46,13 +46,17 @@ export type Props = {
     footerDivider?: boolean;
     appendedContent?: React.ReactNode;
     headerButton?: React.ReactNode;
+    focusableElement?: React.RefObject<HTMLElement>; // a ref to the object you want to set the focus in the modal
```

Comment:
so, the modals that use `GenericModal` will be able to provide a `focusableElement` ref that will get the custom a11y focus event send and set the focus on that focusable Element

Reply:
Instead of passing this in as a dedicated prop, could the modal just define `autoFocus={true}` on one of its elements? I assume there's some differences here related to the `A11yController`, but I know we use `autoFocus` in some other modals